src/benchmark/tpch_data_micro_benchmark.cpp

```diff
@@ -123,6 +125,79 @@ class TPCHDataMicroBenchmarkFixture : public MicroBenchmarkBasicFixture {
   LQPColumnReference _lineitem_orderkey, _lineitem_commitdate, _lineitem_receiptdate;
 };
 
+BENCHMARK_F(TPCHDataMicroBenchmarkFixture, BM_TPCHQ6IndexScan)(benchmark::State& state) {
+  auto& sm = Hyrise::get().storage_manager;
+  auto lineitem_table = sm.get_table("lineitem");
+
+  // Generate an index to benchmark the indexScan
```

Comment:
Guess we can remove this too as it only summarized the next two lines. @XPerianer

(Again later down)